tests/test_losses/test_likelihood.py

```diff
@@ -0,0 +1,74 @@
+# Black-box ABM Calibration Kit (Black-it)
+# Copyright (C) 2021-2022 Banca d'Italia
```

Comment:
we are in 2023, so we should change the year range of the licenses accordingly in all the copyright headers; but that's a task for another day...

Reply:
Right, I will change it for this module only in the meanwhile

Reply:
the copyright check will fail :( , however, you could add the year `2023` among the options in the regex of the script `scripts/check_copyright.py`.

In particular, at line 36, from:

```
# Copyright \(C\) 2021-2022 Banca d'Italia
```

To:

```
# Copyright \(C\) 2021-(2022|2023) Banca d'Italia
```

Reply:
Thanks 🙏🏼